### ISSUE TITLE
`INSTALL_TARGET_PROCESSES` doesn't unload processes on macOS

### ISSUE THREAD

**Issue by jacobcxdev** [theos/theos#515]
`INSTALL_TARGET_PROCESSES` doesn't unload processes when installing SIMBL tweaks with `make do`.

What are the steps to reproduce this issue?
-------------------------------------------
1. Create SIMBL tweak
2. make do

What happens?
-------------
The processes aren't unloaded.

What were you expecting to happen?
----------------------------------
The processes to be unloaded.

Any logs, error output, etc?
----------------------------
Nope.

Any other comments?
-------------------
Nope.

What versions of software are you using?
----------------------------------------
**Operating System:** 10.15.4

**Toolchain Version:** …

**SDK Version:** 10.15.4

**Comment by uroboro** [comment]
Does this happen with any process?

**Comment by jacobcxdev** [comment]
> Does this happen with any process?


@uroboro Yep. I’m pretty sure it was happening because of one of the errors I was getting, because it was magically fixed in #516.
